Makefile.am

```diff
@@ -232,13 +241,13 @@ libusnic_fi_la_SOURCES = $(_usnic_files) $(common_srcs)
 libusnic_fi_la_LDFLAGS = \
         $(usnic_ln_LDFLAGS) \
         -module -avoid-version -shared -export-dynamic
-libusnic_fi_la_LIBADD = $(linkback) $(usnic_nl_LIBS)
+libusnic_fi_la_LIBADD = $(linkback) $(_usnic_libibverbs) $(usnic_nl_LIBS)
```

Comment:
@pmmccorm OIC.  Basically merge what @goodell said above, and instead of using an AM_CONDITIONAL, use an AC_SUBST (a la usnic_nl_LIBS).  Sure, can do.